docs/trace_collection.md

```diff
@@ -18,43 +18,65 @@ Send [traces][1] to Datadog from your Android applications with [Datadog's `dd-s
+   {{< tabs >}} 
+   {{% tab "US" %}}
+   ```kotlin
+    class SampleApplication : Application() {
+        override fun onCreate() {
+            super.onCreate()
+            val configuration = Configuration.Builder().build()
+            val credentials = Credentials(<CLIENT_TOKEN>,<ENV_NAME>,<APP_VARIANT_NAME>,<APPLICATION_ID>)
+            Datadog.initialize(this, credentials, configuration, trackingConsent)
+        }
     }
-}
-```
-
-    {{% /tab %}}
-    {{% tab "EU" %}}
-
-```kotlin
-class SampleApplication : Application() {
-    override fun onCreate() {
-        super.onCreate()
-
-        val config = DatadogConfig.Builder("<CLIENT_TOKEN>", "<ENVIRONMENT_NAME>", "<APPLICATION_ID>")
-                        .useEUEndpoints()
-                        .build()
-        Datadog.initialize(this, trackingConsent, config)
+   ```
+   {{% /tab %}}
+   {{% tab "EU" %}}
+   ```kotlin
+   class SampleApplication : Application() {
+       override fun onCreate() {
+          super.onCreate()
+          val configuration = Configuration.Builder()
+             .useEUEndpoints()
+             .build()
+          val credentials = Credentials(<CLIENT_TOKEN>,<ENV_NAME>,<APP_VARIANT_NAME>,<APPLICATION_ID>)
+          Datadog.initialize(this, credentials, configuration, trackingConsent)
+       }
+   }
+   ```
+   {{% /tab %}} 
+   {{< /tabs >}} 
+   
+   To be compliant with the GDPR regulation, the SDK requires the tracking consent value at initialization. 
+   The tracking consent can be one of the following values:
+    * `TrackingConsent.PENDING`: The SDK starts collecting and batching the data but does not send it to the data
+      collection endpoint. The SDK waits for the new tracking consent value to decide what to do with the batched data.
+    * `TrackingConsent.GRANTED`: The SDK starts collecting the data and sends it to the data collection endpoint.
+    * `TrackingConsent.NOT_GRANTED`: The SDK does not collect any data. You will not be able to manually send any logs, traces, or
+      RUM events.
+    
+   If you want to update the tracking consent after the SDK is initialized, call: `Datadog.setTrackingConsent(<NEW CONSENT>)`.
+   The SDK changes its behavior according to the new consent. For example, if the current tracking consent is `TrackingConsent.PENDING` and you update it to:
+    * `TrackingConsent.GRANTED`: The SDK sends all current batched data and future data directly to the data collection endpoint.
+    * `TrackingConsent.NOT_GRANTED`: The SDK wipes all batched data and does not collect any future data. 
+    
+   You could notice that in the credentials required for initialisation we also require your application variant name. This is very important for us as based on it we will 
+   be able to automatically upload the right proguard `mapping.txt` file at build time. This will be needed to automatically de-obfuscate the stack traces in the Datadog dashboard.
```

Comment:
```suggestion
Note that in the credentials required for initialization, your application variant name is also required. This is important because it enables  the right proguard `mapping.txt` file to be automatically uploaded at build time. This allows a Datadog dashboard to de-obfuscate the stack traces.
```